skycoin/skywire
# Add wintun.dll

Did you run `make format && make check`? Yes

Fixes #	

 Changes:	
- Comment out `GetAvailPublicVPNServers` on gui.go file. Will be use in Systray improvement.
- Add `wintun.dll` to windows msi package.
- Add a checking step before starting vpn-client app for server PK.

How to test this PR:
- Build windows installer by `make win-installer`
- Trying to install it.
- Run `skywire`, in cmd or by its shortcut on start menu.
- Check `C:\Windows\System32` for `wintun.dll`
